kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -287,6 +288,16 @@ export class BrowserInterface {
     }
   }
 
+  public SendAuthentication(data: { rendererAuthenticationType: string }) {
+    let providerType: ProviderType | null = null
+
+    if (data.rendererAuthenticationType === 'wallet_connect') {
+      providerType = ProviderType.WALLET_CONNECT
+    }
+
+    authenticateWhenItsReady(providerType)
+  }
```

Comment:
I think you could do something like this
but we should ask a typescript lord 
@pablitar ? 🙂

```suggestion
  public SendAuthentication(data: { rendererAuthenticationType: string }) {
    const providerType: ProviderType | null = Object.values(ProviderType).includes(data.rendererAuthenticationType as ProviderType) ? data.rendererAuthenticationType as ProviderType : null

    authenticateWhenItsReady(providerType)
  }
```

Reply:
Looks good! Sorry I was late hahaha.